server/src/db/migrations/20230207171703_addUpvotesAndDownVotes.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.table("reviews", table => {
+    table.integer("votes").notNullable().defaultTo(0)
```

Comment:
with voting we are looking for a table in the database. A Vote has an association with the review they are voting on, and we need to know who the user is that is voting

a review has many votes
a user has many votes
a vote belongs to a review
a vote belongs to a user